src/main/java/com/extrahardmode/features/AntiGrinder.java

```diff
@@ -162,7 +162,7 @@ public boolean onEntityDeath(EntityDeathEvent event)
         final boolean inhibitMonsterGrindersEnabled = CFG.getBoolean(RootNode.INHIBIT_MONSTER_GRINDERS, world.getName());
 
         // FEATURE: monsters which take environmental damage or spawn from spawners don't drop loot and exp (monster grinder inhibitor)
-        if (inhibitMonsterGrindersEnabled && entity instanceof Monster && entity.getType() != EntityType.SQUID)
+        if (inhibitMonsterGrindersEnabled && entity instanceof Monster && entity.getType() != EntityType.SQUID && entity.getType() != EntityType.WITHER && entity.getType() != EntityType.GUARDIAN && entity.getType() != EntityType.ELDER_GUARDIAN)
```

Comment:
Probably would make more sense to add these entities to the switch statement and break/return there (since the EntityHelper#isLootLess call may tag such mobspawner-spawned entities and the like).